### ISSUE TITLE
Number of Allied Detachments 

### ISSUE THREAD

**Issue by NewRecruitAnon** [BSData/horus-heresy#3336]
**Problem:**
Warning about too many allied detachements 

**Expected:**
As clarified in the last FAQ you can take as many allied detachments as you wish, as long as they aren’t the same faction.

 **Additional Info:**
See latest FAQ

**Comment by Mayegelt** [comment]
This is one of those pain the in arse ones I think as it can ruin the logic... I am still trying to think of the best way to solve it properly, without having to add in about 40 new categories to the GST... (In essence I could like I did for Legiones Auxilia add in an Allied Force (X) one for each Legion and Force (Such as Mech, Custodes, Sisters, Solar, Militia, DotRS, Bound Daemons...), and then make that have a limit of max 1, and reduce the max limit on it to 0 if a Crusade Force (X) was included for that force (with the exception as detailed before for Mech).
At some point I need to work out how to stop a faction allying with itself, or multiple of the same faction (outside of Mech, who technically with the Archimandrite techno arcana can ally with as many of themselves as they like to get all the bonuses of 11 Archmagos primes if they want, one of every Techno Arcana there is... (12 if you include Draykavac and his unique one...

**Comment by Mayegelt** [comment]
This is still a conflicted issue. The FAQ says:
"Q. Is there a limit on the number of Optional Detachments I can include in my Army?
A. No."
The issue here is that it is not saying definitively that you can have multiple allied detachments. It says you can have multiple "Optional Detachments".
Optional Detachments take many forms. Allied Detachment, Questoris Household & Titan Maniple. Additional to them there is also ones like "Taghma of War Detachment" from the latest campaign books.

**Comment by nstephenh** [comment]
I agree with Maye here. It's only one allied detachment, as many optional detachments as you are given.

**Comment by edvinullman** [comment]
In classic GW fashion there's also this note, from the same FAQ:
> As such, an army whose Primary Detachment is from a given Space Marine Legion may select Allied Detachments of Space Marines as long as each Allied Detachment is from a different Legion (and thus a different Faction) to the Primary Detachment.

This would imply that multiple Allied Detachments are possible. I'm not convinced this is the intention, however, just a classic GW FAQ where the answers just spawns more questions..

**Comment by The4D6** [comment]
> This would imply that multiple Allied Detachments are possible.

The Crusade force organisation chart, as per the core rulebook p281, contains "the Primary Detachment, a Lords of War detachment, and _***an**_ Allied Detachment."

"An" is singular, and the FAQ does not correct, amend, or override this part of the book specifically, it simply clarifies that multiple optional detachments are allowed, not multiple allied.

**Comment by edvinullman** [comment]
Agreed. It’s just very typical that the same document specifying multiple Optional detachments as fine, also does make a notion of multiple Allied detachments.
